docs/blog/2020-02-11-how-impossible-foods-used-gatsby-airtable-to-build-a-store-locator-map/index.md

```diff
@@ -0,0 +1,115 @@
+---
+title: How Impossible Foods Used Gatsby + Airtable to Build A Store Locator Map
+date: 2020-02-11
+author: Sam Bhagwat
+excerpt: "Impossible Foods needed to revamp their website and especially the store locator map functionality to help excited prospective customers find an Impossible Burger at a location near them. The team built the functionality with a whole host of technologies including Gatsby for the frontend, Mapbox for visualization, Airtable to enter store location data and Elastic for natural language search and geolocation filtering."
+tags:
+  - case-studies
+  - ecommerce
+---
+
+In 2018, one of the most promising options in alternative meat, Impossible Foods, was scaling exponentially, but its website was not keeping up. They turned to Gatsby to give their users a beautiful store locator map that helped them solve their most pressing problem i.e., finding an Impossible Burger at a location near them.
+
+## The old website: Failing to help users find an Impossible Burger
+
+[Impossible Foods](https://impossiblefoods.com/) was founded in 2011 by Pat Brown, a Professor Emeritus of Biochemistry at Stanford. Brown created the company not just as an ethical way to eat, but also as a way to combat global warming by reducing emissions due to farm animals. He and the company invented the Impossible Burger, an alternative to meat-based protein.
+
+The first few years of the company were spent mainly in research and iteration, but in 2018 the company was ready to scale. Impossible Foods sold its products through partner restaurants that used Impossible meat-alternative products in their dishes. In 2018, they planned to increase the number of partner restaurants **from 100 to over 5,000** in just one year.
+
+During this time, they found that their website was failing to help excited prospective customers find an Impossible Burger at a location near them. They showed users a static list of all Impossible Food locations, grouped by state.
+
+While this worked at some level, it was difficult for users to find store locations nearby them because they had to scan a long and growing list of locations, sorted alphabetically rather geographically, and then use other tools like Google Maps to figure out their closest location. Many users simply gave up rather than going through this complicated process, causing Impossible Foods to lose excited potential customers.
+
+<Pullquote>
+“Our product would be in certain places, and people wouldn't know. It wouldn’t be in other places, and people would guess that it was there” -- Ashley Geo, Head of Brand Marketing Communications, Impossible Foods.
+</Pullquote>
+
+![Old Maps](./old-maps.gif)
```

Comment:
I think this and `new-maps.gif` below move too quickly. I could see that movement bothering some users and, to my knowledge, there's no way to turn it off on the fly. Can we edit these?